src/mame/layout/connect4.lay

```diff
@@ -217,44 +217,44 @@ license:CC0
 	</element>
 
 	<element name="100p" defstate="0">
-		<text string="100p" state="0">
+		<text string="&#xA3; 1" state="0">
 			<color red="1.0" green="1.0" blue="1.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
-		<text string="100p" state="1">
+		<text string="&#xA3; 1" state="1">
 			<color red="1.0" green="1.0" blue="0.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
 	</element>
 
 	<element name="200p" defstate="0">
-		<text string="200p" state="0">
+		<text string="&#xA3; 2" state="0">
 			<color red="1.0" green="1.0" blue="1.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
-		<text string="200p" state="1">
+		<text string="&#xA3; 2" state="1">
 			<color red="1.0" green="1.0" blue="0.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
 	</element>
 
 	<element name="500p" defstate="0">
-		<text string="500p" state="0">
+		<text string="&#xA3; 5" state="0">
 			<color red="1.0" green="1.0" blue="1.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
-		<text string="500p" state="1">
+		<text string="&#xA3; 5" state="1">
 			<color red="1.0" green="1.0" blue="0.0" />
 			<bounds x="0" y="0.1" width="1" height="0.8" />
 		</text>
 	</element>
 
 	<element name="1000p" defstate="0">
-		<text string="1000p" state="0">
+		<text string="&#xA3; 10" state="0">
```

Comment:
Is there any reason for not just using a £ sign?  All our XML files are UTF-8.

Reply:
Didn't realise that change had been made, last time I checked it wasn't. Will adjust

Reply:
OK, I have a problem, and this may be why I never did this.
I run a srcclean pass on my files before commit (srcclean -u FILE)
If I do that on these, it 'fixes' the unicode character by changing it to a blob, and claims the layouts are ANSI. Is that a problem with my local setup. If so, how best to fix it (I'm running a Windows 10 setup, using the msys2 tools)

Reply:
That’s because you have your text editor set to ISO-8859-1 or Windows 1252 encoding, so it isn’t producing a UTF-8 file.  Fix your editor.

Reply:
notepad++ was eating the encoding, it looks like, anyway when I did the bulk replace I forced UTF-8 parsing and that worked locally,